backend/coach_templates.py

```diff
@@ -116,30 +117,177 @@ def coachExercise(token_claims):
+    max_order = db.session.query(func.max(CoachSession.order)).filter_by(coach_template_id=body['coach_template_id']).scalar()
+    # 
+    # increment the order by 1
+    max_order += 1
+    # create the new session with name, coach_template_id, and order
+    new_session = CoachSession(name=body['name'], coach_template_id=body['coach_template_id'], order=max_order)
+    
+    try:
+        # create new template in CoachTemplate table
+        db.session.add(new_session)
+        db.session.commit()
+    except Exception as e:
+        return {
+            "error": "Internal Server Error"
+        }, 500
+        raise
+
+    # retrieve created session
+    session = CoachSession.query.filter_by(name=body['name']).first()
+
+    result = coach_session_schema.dump(session)
+    
     return result
+
+
+@app.route("/coach/exercise", methods=['POST'])
+@http_guard(renew=True, nullable=False)
+def createCoachExercise(token_claims):
+    if token_claims['role'] != Role.COACH.name:
+        return {
+            "error": "Expected role of COACH"
+        }, 400
+
+    body = request.get_json(force=True)
+    
+    # find current max order value for sexercise belonging to the passed in coach_session_id
+    max_order = db.session.query(func.max(CoachExercise.order)).filter_by(coach_session_id=body['coach_session_id']).scalar()
+    # 
+    # increment the order by 1
+    max_order += 1
+    # create the new coach exercise with coach_exercise_id, coach_session_id, and order
+    new_exercise = CoachExercise(exercise_id=body['exercise_id'], coach_session_id=body['coach_session_id'], order=max_order)
+    
+    try:
+        # create new template in CoachTemplate table
+        db.session.add(new_exercise)
+        db.session.commit()
+    except Exception as e:
+        return {
+            "error": "Internal Server Error"
+        }, 500
+        raise
+
+    # retrieve created exercises
+    exercises = CoachExercise.query.filter_by(coach_session_id=body['coach_session_id'])
+
+    result = coach_exercise_schemas.dump(exercises)
+    
+    return {
+        "session_exercises": result
+    }
+
+@app.route("/exercise", methods=['POST'])
+@http_guard(renew=True, nullable=False)
+def createExercise(token_claims):
+    if token_claims['role'] != Role.COACH.name and token_claims['role'] != Role.CLIENT.name:
+        return {
+            "error": "Expected role of COACH or CLIENT"
+        }, 400
+
+    body = request.get_json(force=True)
+    
+    # create the new exercise with name and category
+    new_exercise = Exercise(name=body['name'], category=body['category'])
+    
+    try:
+        # create new template in CoachTemplate table
+        db.session.add(new_exercise)
+        db.session.commit()
+    except Exception as e:
+        return {
+            "error": "Internal Server Error"
+        }, 500
+        raise
+
+    # retrieve created exercises
+    exercise = Exercise.query.filter_by(name=body['name'], category=body['category']).first()
```

Comment:
I think this should return all exercises just like how `GET /coach/exercise` does since they won't be going into a specific exercise